cfb-mode/src/decrypt.rs

```diff
@@ -21,13 +21,64 @@ where
     iv: Block<C>,
 }
 
+/// CFB mode buffered decryptor.
+#[derive(Clone)]
+pub struct BufferDecryptor<C>
```

Comment:
We could shorten name to `BufDecryptor`. It would be similar to `std::io::BufReader`.

Reply:
fixed